src/main/kotlin/app/revanced/patches/twitter/layout/hideviews/patch/HideViewsBytecodePatch.kt

```diff
@@ -86,7 +87,15 @@ class HideViewsBytecodePatch : BytecodePatch(
 
     private fun removeViewDelegateBinderSubscription() {
         transformMethod(TweetStatsViewDelegateBinderFingerprint) { result, method ->
-            method.removeInstructions(result.scanResult.patternScanResult!!.startIndex - 4, 9)
+            var idx = result.scanResult.patternScanResult!!.startIndex
+            var end = -1
+            var n = 0
+            while (n < 2) {
+                if (method.instruction(idx--).opcode == Opcode.IGET_OBJECT && n++ == 0) {
+                    end = idx
+                }
+            }
+            method.removeInstructions(++idx, end - idx)
```

Comment:
This code is a bit cryptic. The variable `end` is only set once (at the index of the first occurrence of `Opcode.IGET_OBJECT`). The loop continues and defines the range for `method.removeInstructions`. There is a lot going on in a couple of lines which is why it should be simplified. This can be achieved by choosing a different pattern for the fingerprint which also includes the second occurrence of `Opcode.IGET_OBJECT`. After that, a sub-list starting from that index and `indexOfFirst` can be used to determine the end index to remove the instructions.